src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -152,6 +154,26 @@ const ResearchProfile = () => {
             </Typography>
           </Box>
         )}
+        {(!!personBackground || personKeywords.length > 0) && (
+          <Box sx={{ mt: '1rem', display: 'flex', flexDirection: 'column', gap: '0.5rem', width: '70%' }}>
+            <Typography variant="h3">{t('my_page.my_profile.field_and_background.field_and_background')}</Typography>
```

Comment:
Samme her også, eller?
```suggestion
          <>
            <Typography variant="h3" gutterBottom>{t('my_page.my_profile.field_and_background.field_and_background')}</Typography>
```